components/card/index.en-US.md

```diff
@@ -0,0 +1,24 @@
+---
+category: Components
+type: Presentation
+title: Card
+---
+
+Common card container.
+
+## When to use
+
+The most basic card container. You can use it to contain text, lists, pictures and paragraphs. Commonly used in the overview page.
```

Comment:
The most basic card container. You can use it to contain text, lists, pictures and paragraphs. <del>Commonly used in the overview page.</del>